### DISCUSSION TITLE
blog/spring-boot-using-the-new-filter-for-trailing-slash-handling/

### DISCUSSION THREAD

**Discussion by giscus** [LucJosin/lucasjosino.com#31] [Announcements]
# blog/spring-boot-using-the-new-filter-for-trailing-slash-handling/

In this tutorial, we’ll explore how to use the new UrlHandlerFilter to handle trailing slash in Spring Boot 3, including how to forward or redirect requests.

https://www.lucasjosino.com/blog/spring-boot-using-the-new-filter-for-trailing-slash-handling/

**Comment by anton-tkachenko** [comment]
Hey Lucas, thank you for the article. I very much appreciate attaching all the links fo spring issues :)

Can you also please fix the typo in code example? -> I assume, it should be just `@Override`

```
 @Overastro-tDfHKSGh
    protected void doFilterInternal(
```

**Comment by Cuzzie91** [comment]
Great article! However I believe the method signature has changed from version 6.2 onwards. It should now be `UrlHandlerFilter.trailingSlashHandler("/**").
